components/hooks/useEvents.ts

```
@@ -24,95 +24,122 @@ export const useEvents = ({
-      fetcher({
-        page: 0, // Always page 0 for cumulative pagination
-        size, // Increasing size for "load more"
-        place: place !== "catalunya" ? place : undefined,
-        category,
-        byDate: date,
+  const {
+    data: pages,
+    error,
+    isLoading,
+    isValidating,
+    size,
```

ESLint

[error] 'size' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)